codespell_lib/data/dictionary.txt

```diff
@@ -9974,6 +9978,7 @@ invlisible->invisible
 invlove->involve
 invloved->involved
 invloves->involves
+invokable->invocable
 invokation->invocation
 invokations->invocations
 invokee->invoked
```

Comment:
Also suggest invoke, although I'd wonder if invokee could be valid as the thing being invoked, but dictionaries don't seem to agree with it.

Reply:
Done

Reply:
Sorry, this wasn't aimed at you edit, but something I saw a few lines below (hence where the comment was), can you drop it from invokable->invocable and add invokee->invoked, invoke,

Reply:
done